### PR TITLE
Pin conda to 4.13.0

### PR DESCRIPTION
**Pull request by janeyx99**
Recent update to conda 4.14.0 caused breakages in our docker builds:
https://hud.pytorch.org/pytorch/pytorch/commit/754d7f05b6841e555cea5a4b2c505dd9e0baec1d

This pins to prevent the errors:
```
Traceback (most recent call last):
2022-08-24T16:20:49.2412247Z   File "/opt/conda/lib/python3.9/site-packages/conda/exceptions.py", line 1125, in __call__
2022-08-24T16:20:49.2413036Z   File "/opt/conda/lib/python3.9/site-packages/conda/cli/main.py", line 86, in main_subshell
2022-08-24T16:20:49.2413615Z   File "/opt/conda/lib/python3.9/site-packages/conda/cli/conda_argparse.py", line 93, in do_call
2022-08-24T16:20:49.2414282Z   File "/opt/conda/lib/python3.9/site-packages/conda/notices/core.py", line 75, in wrapper
2022-08-24T16:20:49.2415036Z   File "/opt/conda/lib/python3.9/site-packages/conda/notices/core.py", line 39, in display_notices
2022-08-24T16:20:49.2415853Z   File "/opt/conda/lib/python3.9/site-packages/conda/notices/http.py", line 36, in get_notice_responses
2022-08-24T16:20:49.2416661Z   File "/opt/conda/lib/python3.9/site-packages/conda/notices/http.py", line 39, in <genexpr>
2022-08-24T16:20:49.2417399Z   File "/opt/conda/lib/python3.9/concurrent/futures/_base.py", line 609, in result_iterator
2022-08-24T16:20:49.2418145Z   File "/opt/conda/lib/python3.9/concurrent/futures/_base.py", line 446, in result
2022-08-24T16:20:49.2418831Z   File "/opt/conda/lib/python3.9/concurrent/futures/_base.py", line 391, in __get_result
2022-08-24T16:20:49.2419543Z   File "/opt/conda/lib/python3.9/concurrent/futures/thread.py", line 58, in run
2022-08-24T16:20:49.2420292Z   File "/opt/conda/lib/python3.9/site-packages/conda/notices/http.py", line 42, in <lambda>
2022-08-24T16:20:49.2421070Z   File "/opt/conda/lib/python3.9/site-packages/conda/notices/cache.py", line 37, in wrapper
2022-08-24T16:20:49.2421712Z   File "/opt/conda/lib/python3.9/site-packages/conda/notices/http.py", line 58, in get_channel_notice_response
2022-08-24T16:20:49.2422258Z   File "/opt/conda/lib/python3.9/site-packages/requests/sessions.py", line 600, in get
2022-08-24T16:20:49.2422801Z   File "/opt/conda/lib/python3.9/site-packages/requests/sessions.py", line 587, in request
2022-08-24T16:20:49.2423226Z   File "/opt/conda/lib/python3.9/site-packages/requests/sessions.py", line 701, in send
2022-08-24T16:20:49.2423634Z   File "/opt/conda/lib/python3.9/site-packages/requests/adapters.py", line 460, in send
2022-08-24T16:20:49.2424239Z   File "/opt/conda/lib/python3.9/site-packages/requests/adapters.py", line 263, in cert_verify
2022-08-24T16:20:49.2424731Z OSError: Could not find a suitable TLS CA certificate bundle, invalid path: /opt/conda/lib/python3.9/site-packages/certifi/cacert.pem
2022-08-24T16:20:49.2424967Z 
2022-08-24T16:20:49.2425110Z During handling of the above exception, another exception occurred:
2022-08-24T16:20:49.2425279Z 
2022-08-24T16:20:49.2425377Z Traceback (most recent call last):
2022-08-24T16:20:49.2425610Z   File "/opt/conda/bin/conda", line 13, in <module>
2022-08-24T16:20:49.2425845Z     sys.exit(main())
2022-08-24T16:20:49.2426176Z   File "/opt/conda/lib/python3.9/site-packages/conda/cli/main.py", line 129, in main
2022-08-24T16:20:49.2426614Z   File "/opt/conda/lib/python3.9/site-packages/conda/exceptions.py", line 1413, in conda_exception_handler
2022-08-24T16:20:49.2427054Z   File "/opt/conda/lib/python3.9/site-packages/conda/exceptions.py", line 1128, in __call__
2022-08-24T16:20:49.2427555Z   File "/opt/conda/lib/python3.9/site-packages/conda/exceptions.py", line 1170, in handle_exception
2022-08-24T16:20:49.2427995Z   File "/opt/conda/lib/python3.9/site-packages/conda/exceptions.py", line 1181, in handle_unexpected_exception
2022-08-24T16:20:49.2428471Z   File "/opt/conda/lib/python3.9/site-packages/conda/exceptions.py", line 1251, in print_unexpected_error_report
2022-08-24T16:20:49.2428873Z ModuleNotFoundError: No module named 'conda.cli.main_info'
2022-08-24T16:20:55.5428691Z The command '/bin/sh -c bash ./install_conda.sh && rm install_conda.sh' returned a non-zero code: 1
```

### CODE REVIEW

File: .circleci/docker/common/install_conda.sh [hunk 1]
```diff
@@ -55,6 +55,10 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
   # Ensure we run conda in a directory that jenkins has write access to
   pushd /opt/conda
 
+  # Prevent conda from updating to 4.14.0, which causes docker build failures
+  # Examples: https://hud.pytorch.org/pytorch/pytorch/commit/754d7f05b6841e555cea5a4b2c505dd9e0baec1d
+  echo "conda ==4.13.0" >> ~/conda-meta/pinned
+
   # Track latest conda update
   as_jenkins conda update -y -n base conda
```
Comment (malfet) [inline .circleci/docker/common/install_conda.sh:63]: Err, why updating it at all then? Just use the bundled one...
```suggestion
  #  Use conda 4.13.0, as 4.14. causes docker build failures,
  #  See https://hud.pytorch.org/pytorch/pytorch/commit/754d7f05b6841e555cea5a4b2c505dd9e0baec1d
  # Uncomment when resolved
  # as_jenkins conda update -y -n base conda
```

**Review by malfet — APPROVED** [review]

File: .circleci/docker/common/install_conda.sh (hunk 1 above)
Comment (janeyx99) [reply .circleci/docker/common/install_conda.sh:63]: Cuz the prev conda installed is 4.12, but yea, that's a simpler change, will take suggestion.

**Pull request closed without merging**